cql3/Cql.g

```diff
@@ -1915,6 +1919,8 @@ K_PARTITION:   P A R T I T I O N;
 
 K_SCYLLA_TIMEUUID_LIST_INDEX: S C Y L L A '_' T I M E U U I D '_' L I S T '_' I N D E X;
 K_SCYLLA_COUNTER_SHARD_LIST: S C Y L L A '_' C O U N T E R '_' S H A R D '_' L I S T; 
+K_SCYLLA_CLUSTERING_BOUND: S C Y L L A '_' C L U S T E R I N G '_' B O U N D;
+
```

Comment:
Add to unreserved keyword list?

Reply:
Perhaps no need, since the SCYLLA_ prefix is sort of reserved.

Reply:
We did not add the other scylla_* things as keywords, so did not do here either.